auth/clerk_client.go

```diff
@@ -86,6 +88,11 @@ func (h ClerkHandler) Session(next echo.HandlerFunc) echo.HandlerFunc {
 		c.Request().Header.Set(echo.HeaderAuthorization, fmt.Sprintf("Bearer %s", token))
 		c.Request().Header.Set(api.UserIDHeaderKey, user.ID.String())
 
+		trace.SpanFromContext(c.Request().Context()).SetAttributes(
```

Comment:
Why not just add a ctx.WithAttribute[s] function  and/or update WithUser to set the the trace attributes as well

Reply:
👍🏽